ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -25,14 +25,27 @@ import scala.util.{Failure, Success, Try}
 
 object Main {
 
-  private[this] val logger = ContextualizedLogger.get(getClass)
+  private[this] lazy val logger = ContextualizedLogger.get(getClass)
```

Comment:
Why making this lazy? If it's not used it can simply be removed, if it's used what's to gain from deferring its creation?

Reply:
Good catch, it's not necessary anymore. In a prior version I had to defer it's creation so the configuration for logback is proceed late enough such that the changed system properties are used.